modules/nw-ptp-operator-metrics-reference.adoc

```diff
@@ -22,54 +22,78 @@ Some of the following metrics are applicable for PTP grandmaster clocks (T-GM) o
 
 |`openshift_ptp_clock_class`
 |Returns the PTP clock class for the interface.
-Possible values for PTP clock class are `6` (`LOCKED`), `7` (`HOLDOVER` within specification), `140` (`HOLDOVER` outside specification), and `248` (`FREERUN`).
+Possible values for PTP clock class are 6 (`LOCKED`), 7 (`PRC UNLOCKED IN-SPEC`), 52 (`PRC UNLOCKED OUT-OF-SPEC`), 187 (`PRC UNLOCKED OUT-OF-SPEC`), 248 (`DEFAULT`), or 255 (`SLAVE ONLY CLOCK`).
 Applicable to T-GM clocks only.
-|`openshift_ptp_clock_class {node="compute-1.example.com", process="ptp4l"} 6`
+|`{node="compute-1.example.com",process="ptp4l"} 6`
 
 |`openshift_ptp_clock_state`
 |Returns the current PTP clock state for the interface.
 Possible values for PTP clock state are `FREERUN`, `LOCKED`, or `HOLDOVER`.
-|`openshift_ptp_clock_state {iface="CLOCK_REALTIME", node="compute-1.example.com", process="phc2sys"} 1`
+|`{iface="CLOCK_REALTIME", node="compute-1.example.com", process="phc2sys"} 1`
 
 |`openshift_ptp_delay_ns`
 |Returns the delay in nanoseconds between the primary clock sending the timing packet and the secondary clock receiving the timing packet.
-|`openshift_ptp_delay_ns {from="master", iface="ens2fx", node="compute-1.example.com", process="ts2phc"} 0`
+|`{from="master", iface="ens2fx", node="compute-1.example.com", process="ts2phc"} 0`
 
 |`openshift_ptp_frequency_adjustment_ns`
 |Returns the frequency adjustment in nanoseconds between 2 PTP clocks.
 For example, between the upstream clock and the NIC, between the system clock and the NIC, or between the PTP hardware clock (`phc`) and the NIC.
 Applicable to T-GM clocks only.
-|`openshift_ptp_frequency_adjustment_ns {from="phc", iface="CLOCK_REALTIME", node="compute-1.example.com", process="phc2sys"} -6768`
+|`{from="phc", iface="CLOCK_REALTIME", node="compute-1.example.com", process="phc2sys"} -6768`
+
+|`openshift_ptp_frequency_status`
+|Returns the current status of the DPLL frequency for the NIC.
+Possible values are -1 (`UNKNOWN`), 0 (`INVALID`), 1 (`FREERUN`), 2 (`LOCKED`), 3 (`LOCKED_HO_ACQ`), or 4 (`HOLDOVER`).
+|`{from="dpll",iface="ens2fx",node="compute-1.example.com",process="dpll"} 3`
+
+|`openshift_ptp_phase_status`
+|Returns the status of the DPLL phase for the NIC.
```

Comment:
Does it make sense to mention this is _Digital Phase-Locked Loop_ at first usage here?